app/routes/about.tsx

```diff
@@ -1,30 +1,13 @@
 import type { V2_MetaFunction } from "@remix-run/react";
 
 import { PageGrid } from "~/components/PageGrid";
-
-const team = [
-	{
-		name: "Joe Woods",
-		role: "Director",
-	},
-	{
-		name: "Chasen Stark",
-		role: "Events",
-	},
-	{
-		name: "Josh Goldberg",
-		role: "Marketing",
-	},
-	{
-		name: "Cody Lewis",
-		role: "Community",
-	},
-];
+import team from "~/data/team.json";
+import { constructSiteTitle } from "~/utils/common";
 
 const organizersEmeritum = ["Rasha Moumneh", "Matt Brophy"];
```

Comment:
[Refactor] Think we should also put `organizersEmeritum` into the data place, along with `team`?

Reply:
Maybe separate this file into "organizers" and "organizersEmeritum"

Reply:
Updated, also now using a loader to make typescript and remix happy :)